src/ch/imedias/rsccfx/localization/Bundle_de.properties

```diff
@@ -38,34 +38,34 @@ homeSupportBigLbl=Ich m\u00F6chte helfen
 homeSupportSmallLbl=Jemand ben\u00F6tigt meine Hilfe
 
 # RequestView
-requestTitleLbl=Schl\u00fcssel generieren
-requestDescriptionLbl=W\u00e4hlen Sie den Supporter, welcher Sie unterst\u00fctzen soll. Sie k\u00F6nnen auch neue Supporter hinzuf\u00fcgen oder einen bestehenden editieren. Kontaktieren Sie Ihren Supporter f\u00fcr die Einstellungsdetails.
-requestKeyGeneratorPane=Schl\u00fcsselgenerierung
-requestGeneratedKeyFld=Schl\u00fcssel wird generiert...
+requestTitleLbl=Schl\u00FCssel generieren
+requestDescriptionLbl=W\u00E4hlen Sie den Supporter, welcher Sie unterst\u00FCtzen soll. Sie k\u00F6nnen auch neue Supporter hinzuf\u00FCgen oder einen bestehenden editieren. Kontaktieren Sie Ihren Supporter f\u00FCr die Einstellungsdetails.
+requestKeyGeneratorPane=Schl\u00FCsselgenerierung
+requestGeneratedKeyFld=Schl\u00FCssel wird generiert...
 requestPredefinedAdressessPane=Adressbuch
 requestStatusLblWaiting=Warte...
 requestStatusLblReady=Bereit
-requestSupporterDescriptionLbl=W\u00e4hlen Sie den Supporter, welcher Sie unterst\u00fctzen soll. Sie k\u00F6nnen auch neue Supporter hinzuf\u00fcgen oder einen bestehenden editieren. Kontaktieren Sie Ihren Supporter f\u00fcr die Einstellungsdetails.
+requestSupporterDescriptionLbl=W\u00E4hlen Sie den Supporter, welcher Sie unterst\u00FCtzen soll. Sie k\u00F6nnen auch neue Supporter hinzuf\u00FCgen oder einen bestehenden editieren. Kontaktieren Sie Ihren Supporter f\u00FCr die Einstellungsdetails.
 
 # SupportView
-supportTitleLbl=Schl\u00fcssel eingeben
-supportDescriptionLbl=Initialisiert eine neue Fernwartung durch die Eingabe des generierten Schl\u00fcssels Ihres Partners
+supportTitleLbl=Schl\u00FCssel eingeben
+supportDescriptionLbl=Initialisiert eine neue Fernwartung durch die Eingabe des generierten Schl\u00FCssels Ihres Partners
 supportConnectBtn=Verbinden
-supportKeyInputPane=Schl\u00fcsseleingabe
+supportKeyInputPane=Schl\u00FCsseleingabe
 supportAdressBookPane=Dienst starten
 supportStatusLblWaiting=Warte...
 supportStatusLblReady=Bereit
 
 #StartServiceView
 startServiceDescpriptionLbl=Damit Ihr Partner durch das Adressbuch eine direkte Verbindung zu Ihnen herstellen kann, muss hier zuerst der Dienst gestartet werden.
-statusBoxServiceIdle=Dienst l�uft nicht
+statusBoxServiceIdle=Dienst l\uFFFDuft nicht
```

Comment:
search all of the bundle files for "\uFFFD" it will reveal all of the ? symbols => http://www.fileformat.info/info/unicode/char/0fffd/index.htm
there are quite a lot of them unfortunately...